src/cli/MeasureUpdateCommand.cpp

```diff
@@ -88,6 +97,203 @@ namespace cli {
+                                                                               opt->newMeasureOpts.measureLanguage.valueName(), measureLanguageNames))
+                                                      ->option_text("measureLanguage")
+                                                      ->check(CLI::IsMember(measureLanguageNames));
+      }
+
+      newMeasureSubCommand->add_option("-c,--class-name", opt->newMeasureOpts.className, "Measure Class Name [Required]")
+        ->option_text("className")
+        ->required(true);
+
+      newMeasureSubCommand->add_option("-n,--name", opt->newMeasureOpts.name, "Measure Human Readable Name [Default: className]")
+        ->option_text("name")
+        ->required(false);
+
+      std::vector<std::string> taxonomyTags;
+      std::vector<std::string> firstLevelTaxonomyTerms = openstudio::BCLMeasure::suggestedFirstLevelTaxonomyTerms();
+      taxonomyTags.reserve(35);  // 2023-05-03: at this moment, this is the total number
+      for (auto& firstLevelTaxonomyTerm : firstLevelTaxonomyTerms) {
+        auto secondLevelTaxonomyTerms = openstudio::BCLMeasure::suggestedSecondLevelTaxonomyTerms(firstLevelTaxonomyTerm);
+        taxonomyTags.reserve(taxonomyTags.size() + secondLevelTaxonomyTerms.size());
+        std::transform(secondLevelTaxonomyTerms.begin(), secondLevelTaxonomyTerms.end(), std::back_inserter(taxonomyTags),
+                       [&firstLevelTaxonomyTerm](auto&& secondLevelTaxonomyTerm) {
+                         return fmt::format("{}.{}", firstLevelTaxonomyTerm, secondLevelTaxonomyTerm);
+                       });
+      }
+
+      newMeasureSubCommand
+        ->add_option("--taxonomy-tag", opt->newMeasureOpts.taxonomyTag, fmt::format("Taxonomy Tag [Default: '{}']", opt->newMeasureOpts.taxonomyTag))
+        ->option_text("tag")
+        ->required(false)
+        ->check(CLI::IsMember(taxonomyTags))
+        ->capture_default_str();
+      newMeasureSubCommand->add_option("-d,--description", opt->newMeasureOpts.description, "Description")->option_text("desc")->required(false);
+      newMeasureSubCommand->add_option("-m,--modeler-description", opt->newMeasureOpts.modelerDescription, "Modeler Description")
+        ->option_text("modeler_desc")
+        ->required(false);
+
+      [[maybe_unused]] auto* directoryPathOpt =
+        newMeasureSubCommand->add_option("DIRECTORY", opt->newMeasureOpts.directoryPath, "Directory for the measure")
+          ->option_text(" ")
+          ->required(true)
+          ->check(CLI::NonexistentPath);
+
+      {
+        static constexpr auto helpOptionsGroupName = "Help";
+        static constexpr auto newMeasureExamples = R"(Examples:
+
+1. Create a Ruby ModelMeasure:
+
+```
+$ openstudio labs measure new --class-name MyExampleRubyModelMeasure
+$ openstudio labs measure new --class-name MyExampleRubyModelMeasure --type ModelMeasure --language Ruby
+```
+
+2. Pass all optional args to create a Python EnergyPlusMeasure:
+
+```
```

Comment:
Helpers, because I feel like this is going to be more than helpful given the number of options we can optionally pass